src/app/services/tasks.service.ts

```diff
@@ -25,4 +25,17 @@ export class TasksService {
       `https://63761992b5f0e1eb850298da.mockapi.io/tasks/${taskId}`
     );
   }
+
+  editTaskById(categoryId: string, task: TaskModel): Observable<TaskModel> {
```

Comment:
update or edit. You are in task service so you don't need to add it to name of the method